CartMgmt/Cart/cart.css

```diff
@@ -0,0 +1,84 @@
+@import url(https://fsharpui.netlify.app/main.css);
+* {
```

Comment:
These lines of code can be part of reset or base css.

Reply:
Will look into it.